Source/WebCore/editing/Editor.cpp

```diff
@@ -3691,6 +3693,8 @@ void Editor::removeTextPlaceholder(TextPlaceholderElement& placeholder)
     // To match the Legacy WebKit implementation, set the text insertion point to be before where the placeholder used to be.
     if (document->selection().isFocusedAndActive() && document->focusedElement() == savedRootEditableElement)
         document->selection().setSelection(VisibleSelection { savedPositionBeforePlaceholder }, FrameSelection::defaultSetSelectionOptions(UserTriggered::Yes));
+
+    document->selection().setCaretVisible(true);
```

Comment:
Similarly if the selection is not focused/active, this may cause us to incorrectly start showing the caret.